client/src/actors/client.rs

```diff
@@ -27,9 +27,63 @@ use core::{
 };
 use std::{path::PathBuf, time::Duration};
 
+use zeroize::Zeroizing;
+
 #[cfg(feature = "communication")]
 use communication::actor::{PermissionValue, RequestPermissions, ToPermissionVariants, VariantPermission};
 
+pub type SLIP10Chain = Vec<u32>;
+
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+pub enum SLIP10Curve {
+    Ed25519,
+    Secp256k1,
+}
+
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+pub enum Secp256k1EcdsaFlavor {
+    Keccak256,
+    Sha256,
+}
+
+pub mod serde_bip39_mnemonic {
+    use super::bip39;
+
+    pub fn serialize<S>(m: &bip39::Mnemonic, s: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        s.serialize_str(m.as_ref())
+    }
+
+    pub fn deserialize<'de, D>(d: D) -> Result<bip39::Mnemonic, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        use serde::Deserialize;
+        String::deserialize(d).map(bip39::Mnemonic::from)
+    }
+}
+
+pub mod serde_bip39_passphrase {
```

Comment:
Can you use only one like in the other PR?

Reply:
Forgot to update it. Fixed.